chrome/content/zotero/xpcom/quickCopy.js

```diff
@@ -25,27 +25,61 @@
 
 
 Zotero.QuickCopy = new function() {
-	this.getFormattedNameFromSetting = getFormattedNameFromSetting;
-	this.getSettingFromFormattedName = getSettingFromFormattedName;
-	this.getContentType = getContentType;
-	this.stripContentType = stripContentType;
-	this.getFormatFromURL = getFormatFromURL;
-	this.getContentFromItems = getContentFromItems;
 	
 	var _initialized = false;
 	var _formattedNames = {};
 	
+	/*
+	 * Return Quick Copy setting object from string, stringified object, or object
+	 * 
+	 * Example string format: "bibliography/html=http://www.zotero.org/styles/apa"
+	 *
+	 * Quick Copy setting object has the following properties:
+	 * - "mode": "bibliography" (for styles) or "export" (for export translators)
+	 * - "contentType: "" (plain text output) or "html" (HTML output; for styles
+	 *   only)
+	 * - "id": style ID or export translator ID
+	 * - "locale": locale code (for styles only)
+	 */
+	this.unserializeSetting = function (setting) {
+		var settingObject = {};
+		
+		if (typeof setting === 'string' || setting instanceof String) {
```

Comment:
I just used the code from http://stackoverflow.com/a/7772724/1712389. Didn't know whether the second test was necessary.